packages/color-block/src/components/TooltipContent.spec.ct.tsx

```diff
@@ -0,0 +1,41 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from 'cypress/react';
+
+import { TooltipContent } from './TooltipContent';
+
+const TooltipContentSelector = '[data-test-id="tooltip-content"]';
+
+const COLOR_VALUE = '#ff0000';
+const STATUS_IDLE_MESSAGE = 'Click to copy.';
+const STATUS_SUCCESS_MESSAGE = 'Copied!';
+const STATUS_ERROR_MESSAGE = 'Error copying. Try again.';
+
+describe('TooltipContent component', () => {
+    it('renders a tooltip content component with idle status', () => {
+        mount(<TooltipContent colorValue={COLOR_VALUE} status="idle" />);
+
+        cy.get(TooltipContentSelector).should('exist');
```

Comment:
Nit: I think you could omit the `.should('exist')` check here (and also in some other tests), because `.contains('Color Name')` (next line) would also fail if the element doesn't exist.

Reply:
Removed all instances I found unnecessary

Reply:
Nice, thanks!